resources/master_crosswalk/info/SPModelInfo.md

```diff
@@ -15,4 +15,214 @@ output:
 
 | Variable name | iText |
 |--------------:|:------|
-|        test|blah|
+|**GMH** | **UI Variables**|
+GMH Care Quality|Quality care has many dimensions, but in this model we are mainly concerned with how quality impacts the time it takes patients' symptoms to improve; specifically impacting the variables "Time to Improve" and "Time to Unflag."  A shorter time implies faster recovery. 
```

Comment:
Does "quality impact the time it takes..." or do we mean
"we are mainly concerned with quality in terms of the time it takes patients' symptoms..."
OR is the time to improve an actual measure, thus
"we are mainly concerned with quality as measured by Time to Improve and Time to Unflag - indicators of the speed of patients' improvement"

Reply:
We don't have a **direct** estimate of "Time to Improve", as "improve" is based not on patients' actual symptoms, but on where they end up (ie, if they are eventually stepped up or down, where being stepped down is a clear signal that their symptoms have "improved"). That distinction should be in the itext for the data table, and in the ModelParameters file.  We do have a direct estimate on the "Time to UnFlag."  
So, how, I guess, we should go with "we are mainly concerned with quality in terms of the time it takes patients' symptoms..."